hazelcast/src/test/java/com/hazelcast/instance/DefaultAddressPickerTest.java

```diff
@@ -232,6 +237,57 @@ public void testPublicAddress_withInvalidAddress() throws Exception {
         addressPicker.pickAddress();
     }
 
+    @Test
+    public void testEqualsAndHashCode() throws Exception {
+        InterfaceDefinition interfaceDefinition = new InterfaceDefinition("localhost", "127.0.0.1");
```

Comment:
Couldn't get the `EqualsVerifier` running with the existing setup of class and subclass. So I've added an old-style `equals()` and `hashcode()` test for both classes.